.gitignore

```diff
@@ -1,27 +1,106 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
Same comments as previously still apply here.

Reply:
Unless you have reason to believe that dulwich will generate these files, they should not be in here.

Reply:
OK. If I think it can be deleted, I will delete it, but what I refuse will be what I think I want to keep